config/shiny-server-rules.config

```diff
@@ -195,3 +195,10 @@ log_as_user {
   at $ server location;
   maxcount 1;
 }
+
+disable_reconnect {
+  desc "When a user's connection to the server is interrupted, Shiny Server will offer them a dialog that allows them to reconnect to their existing Shiny session for 15 seconds. This implies that the server will keep the Shiny session active on the server for an extra 15 seconds after a user disconnects in case they reconnect. After the 15 seconds, the user's session will be reaped and they will be notified and offered an opportunity to refresh the page. If this setting is true, the server will immediately reap the session of any user who is disconnected.";
```

Comment:
We don't have to. The internal implementation isn't aware of the delay because we hold back the socket's `close` and `end` events for 15s. So they assume the connection is still live until 15s after it died.